bundles/org.openhab.binding.senechome/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -101,6 +92,10 @@
 			<channel id="wallbox1ChargingPower" typeId="wallbox1ChargingPower"/>
 		</channels>
 
+		<properties>
+			<property name="thingTypeVersion">2</property>
```

Comment:
```suggestion
			<property name="thingTypeVersion">1</property>
```

Reply:
@jlaur i thought the idea is that the current one (on main) is version 1 and the new one is version 2.
In the update/instructions.xml file is defined that 7 channels are removed for version 1.

Reply:
No, main has no version, So this version 1 is the first change recorded.

Reply:
@lsiepel  is right.